LibTest/ffi/UnsignedLongLong/UnsignedLongLong_A01_t01.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion The C unsigned long long type.
+///
+/// Typically an unsigned 64-bit integer. For a guaranteed 64-bit integer, use
+/// [Uint64] with the C uint64_t type. For a signed long long, use [LongLong].
+///
+/// The [UnsignedLongLong] type is a native type, and should not be constructed
+/// in Dart code. It occurs only in native type signatures and as annotation on
+/// [Struct] and [Union] fields.
+///
+/// @description Checks that this type represents C unsigned long long type
+/// @author sgrekhov22@gmail.com
+
+import "dart:ffi";
+import "package:ffi/ffi.dart";
+import "../../../Utils/expect.dart";
+
+void main() {
+  Pointer<UnsignedLongLong> p1 = calloc<UnsignedLongLong>();
+  try {
+    p1.value = 42;
+    Expect.equals(42, p1.value);
+    p1.value = -42;
+    Expect.equals(-42, p1.value);
```

Comment:
I believe that it is expected that `UnsignedLongLong` value is negative (because we have no Unsigned 64-bit types in Dart to accomodate `unsigned long long` value). @dcharkes please confirm

Reply:
correct